src/interpreter-errors.ts

```diff
@@ -1,4 +1,9 @@
-/* tslint:disable:max-classes-per-file */
+/*
```

Comment:
Not true. InvalidNumberOfArguments is not a JS error but a Source one for example. Should this mean that we should try to separate these away from the JS ones?